### DISCUSSION TITLE
Cleanup Attachments missing?

### DISCUSSION THREAD

**Discussion by naquad** [zim-desktop-wiki/zim-desktop-wiki#1855] [Q&A] [answered]
Hello,

I'm using ZIM 0.74.3.

In accordance with the bundled manual and also the [online one](https://zim-wiki.org/manual/Help/Menu_Items.html) there should be a menu entry Tools->Cleanup Attachments that would allow me to remove attachments not referenced by any pages.
That point is missing, even code has no mentions of any similar functionality.

Where did it go and how do I cleanup now?

Thank you.

**Comment by introt** [comment]
No idea why that's in the manual - it has not been implemented yet. Feel free to follow issue #1584 for the latest developments.

**Comment by jaap-karssenberg** [comment, marked as answer]
Some digging in the log showed a dialog being merged in 2011 and deleted
again in 2014. Afraid I didn't log the reason for the removal; most likely
it was not maintained and / or buggy at the time.

With the removal in 2014 the manual should have been updated as well. Will
fix that.
